worker/uniter/modes.go

```diff
@@ -302,6 +306,13 @@ func modeAbideAliveLoop(u *Uniter) (Mode, error) {
 			time.Now(), lastCollectMetrics, metricsPollInterval,
 		)
 
+		lastSentMetrics := time.Unix(u.operationState().SendMetricsTime, 0)
+		logger.Errorf("metric send interval: %v", metricsSendInterval)
```

Comment:
Is Errorf an appropriate log level for these messages? They look like Debugf-level content to me.